### PR TITLE
Improving readme

### PR DESCRIPTION
**Pull request by csokol**
please check my english :-)

### CODE REVIEW

**Comment by csokol** [comment]
actually, the vraptor-environment property is `mailer.implementation`, not `MAILER_IMPLEMENTATION` as i wrote

File: README-pt-br.markdown [hunk 1]
```diff
@@ -163,6 +163,12 @@ Caso seja conveniente, configure ainda no arquivo de propriedades de cada ambien
 do servidor de envio de e-mail:
 	vraptor.simplemail.main.replyTo = support@myapp.com
 
+No ambiente 'development', o vraptor-simplemail irá usar o MockMailer simular o envio dos emails. Essa
```
Comment (luiz) [inline README-pt-br.markdown]: - irá usar => usará
- faltou um "para" antes do simular, não?

File: README-pt-br.markdown [hunk 2]
```diff
@@ -163,6 +163,12 @@ Caso seja conveniente, configure ainda no arquivo de propriedades de cada ambien
...
+implementação simplesmente loga o envio dos emails com o sl4j.
+
+Em qualquer outro ambiente, o vraptor-simplemail irá usar o DefaultMailer ou a classe que pode
```
Comment (luiz) [inline README-pt-br.markdown]: irá usar => usará

**Comment by luiz** [comment]
English is ok for me! But pt-BR needs some corrections. And, please, correct the `MAILER_IMPLEMENTATION` you pointed ;)

**Comment by csokol** [comment]
done!

**Comment by luiz** [comment]
Merged. Thanks!

**Pull request merged**